src/ui/components/modals/AppModal.tsx

```diff
@@ -201,7 +203,9 @@ export const AppModal: FunctionComponent<Props> = ({
           </StyledScrollView>
         ) : (
           <React.Fragment>
-            <SpacerBetweenHeaderAndContent />
+            {!shouldRemoveSpacerBetweenHeaderAndContent && (
+              <SpacerBetweenHeaderAndContent testID="spacerBetweenHeaderAndContent" />
+            )}
```

Comment:
Ici tu aurais donc ça

```suggestion
            {addSpacerBetweenHeaderAndContent && (
              <SpacerBetweenHeaderAndContent testID="spacerBetweenHeaderAndContent" />
            )}
```